.husky/pre-commit

```diff
@@ -0,0 +1 @@
+yarn pretty-quick
```

Comment:
```suggestion
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"

npx pretty-quick --staged
```

The `pretty-quick` script can also be removed

Reply:
Done but used yarn since repo use that.